crates/rolldown/tests/rolldown/issues/6880/main.js

```diff
@@ -0,0 +1,4 @@
+exports.foo = function a() {
```

Comment:
Is it possible to use `toString` to add an assertion here? Just curious, I remember toString could return the source code.

Reply:
done